hat/app/org/hatdex/hat/utils/Mailer.scala

```diff
@@ -123,103 +120,91 @@ class HatMailerImpl @Inject() (
     val mailerClient: MailerClient
   )(implicit ec: RemoteExecutionContext)
     extends HatMailer {
-  private val emailFrom = configuration.get[String]("play.mailer.from")
+  private val emailFrom   = configuration.get[String]("play.mailer.from")
   private val adminEmails = configuration.get[Seq[String]]("exchange.admin")
 
   def serverErrorNotify(
       request: RequestHeader,
       exception: UsefulException
-    )(implicit m: Messages
-    ): Done = {
+    )(implicit m: Messages): Done = {
     sendEmail(adminEmails: _*)(
       from = emailFrom,
       subject = s"HAT server ${request.host} error #${exception.id}",
       bodyHtml = views.html.mails.emailServerError(request, exception),
-      bodyText =
-        views.html.mails.emailServerError(request, exception).toString()
+      bodyText = views.html.mails.emailServerError(request, exception).toString()
     )
     Done
   }
 
   def serverExceptionNotify(
       request: RequestHeader,
       exception: Throwable
-    )(implicit m: Messages
-    ): Done = {
+    )(implicit m: Messages): Done = {
     sendEmail(adminEmails: _*)(
       from = emailFrom,
       subject =
         s"HAT server ${request.host} error: ${exception.getMessage} for ${request.path + request.rawQueryString}",
       bodyHtml = views.html.mails.emailServerThrowable(request, exception),
-      bodyText =
-        views.html.mails.emailServerThrowable(request, exception).toString()
+      bodyText = views.html.mails.emailServerThrowable(request, exception).toString()
     )
     Done
   }
 
+  // add MessagesApi implicitly
+  // remove Messages
+  // change the txt and html email templates
+  // emailAuthPasswordChange {txt|html}
   def passwordReset(
       email: String,
-      user: HatUser,
       resetLink: String
-    )(implicit m: Messages,
-      server: HatServer
-    ): Done = {
+    )(implicit m: MessagesApi,
+      lang: Lang,
+      server: HatServer): Done = {
     sendEmail(email)(
       from = emailFrom,
       subject = s"HAT ${server.domain} - reset your password",
```

Comment:
The same as password changed template, we will need to get the subject from the messages.

Reply:
Yup, waiting for messages